components/motor/gpio/motor_encoder_ramp_test.go

```diff
@@ -2,34 +2,33 @@ package gpio
 
 import (
```

Comment:
can I delete this file? I don't see `computeRamp` coming back but even if it does I feel like that could be in the regular test file

Reply:
agree to delete.